plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/config.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from typing import Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+
+
+@dataclass
+class RedisConfEnv:
+    host: str = "${env:REDIS_HOST}"
+    port: int = "${env:REDIS_PORT}"
+    db: Optional[str] = "${env:REDIS_DB}"
+    password: Optional[str] = "${env:REDIS_PASSWORD}"
```

Comment:
This passes mypy just because your interpolated values are accidentally strings.
get used to using SI or II ([doc](https://omegaconf.readthedocs.io/en/latest/structured_config.html#interpolations).)